core/chain-configs/src/test_genesis.rs

```diff
@@ -492,7 +522,7 @@ impl TestGenesisBuilder {
             validators: derived_validator_setup.validators,
             num_block_producer_seats: derived_validator_setup.num_block_producer_seats,
             num_chunk_only_producer_seats: 0,
-            minimum_stake_ratio: derived_validator_setup.minimum_stake_ratio,
```

Comment:
Doesn't that change the min ratio for existing tests that used derived validator setup?Wouldn't it be better to use the derived ratio when the user doesn't explicitly provide one instead of setting it to zero? Changing the ratio might break some existing tests (although it probably doesn't).

Reply:
Ah nevermind, I missed the part that modifies derived setup